docs/.images/diagrams/uds-core-operator-authservice-keycloak.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Definitely a nit, but comment on the diagram - the `is enableAuthserviceSelector present` "conditional" is really only applicable if `spec.sso` is present (it's actually a field under that). This diagram makes it seem like you can have authservice without SSO. I'm bad at diagram organization, but it would be good to represent that the authservice check ONLY happens after a client creation (after storing registration token) and cannot happen if there is no spec.sso.

Reply:
Great spot! Updated!